src/components/home.js

```diff
@@ -114,6 +114,7 @@ function Home(props) {
                 <div
                   className={`tab ${graphOption === 1 ? 'focused' : ''}`}
                   onClick={() => {
+                    console.log(graphOption);
```

Comment:
Please can you remove console log

Reply:
Hi,
I have removed console, if you want to pull, i will merge with my master you can pull from there.